src/plugin/Manager.cc

```diff
@@ -883,26 +888,31 @@ void Manager::HookUpdateNetworkTime(double network_time) const
 	}
 
 void Manager::HookBroObjDtor(void* obj) const
+	{
+	HookObjDtor(obj);
+	}
+
+void Manager::HookObjDtor(void* obj) const
 	{
 	HookArgumentList args;
 
 	if ( HavePluginForHook(META_HOOK_PRE) )
 		{
 		args.push_back(HookArgument(obj));
-		MetaHookPre(HOOK_BRO_OBJ_DTOR, args);
+		MetaHookPre(HOOK_OBJ_DTOR, args);
 		}
 
-	hook_list* l = hooks[HOOK_BRO_OBJ_DTOR];
+	hook_list* l = hooks[HOOK_OBJ_DTOR];
 
 	if ( l )
 		for ( hook_list::iterator i = l->begin(); i != l->end(); ++i )
 			{
 			Plugin* p = (*i).second;
-			p->HookBroObjDtor(obj);
+			p->HookObjDtor(obj);
```

Comment:
I believe we need to call both here, otherwise the old one will not execute at all any more.

Actually, this together with the note above lets me wondering if we need to do this differently and treat the two as completely separate hooks that are just both triggered at the same time (inside the Obj destructor), even if that means code duplication.

Reply:
That'd be easy enough to do, and to mark the old version of the request methods as deprecated.

Reply:
I implemented this, and rebased the whole thing on `master` again.